tests/core/rke_dind.py

```diff
@@ -0,0 +1,54 @@
+from jinja2 import Template
+import random
+import subprocess
+
+
+def generate_rke_clsuter_config(dind_rke_node_num):
+    # generate a radnom and kube_config file
+    dind_random = random.randint(1, 100000)
+    dind_cluster_config_file = "dind-" + str(dind_random) + ".yml"
+    dind_kube_config_file = "dind-" + str(dind_random) + "_kube_config.yml"
+    cluster_config_tmpl = get_rke_config_template()
+
+    # generate nodes
+    random_nodes = [
+            "node-" +
+            str(random.randint(10000, 99999))
+            for x in range(dind_rke_node_num)]
+    rke_config_template = Template(cluster_config_tmpl)
+    rendered_tmpl = rke_config_template.render(
+        random_nodes=random_nodes)
+
+    # write config file on disk
+    cluster_config_file = open(dind_cluster_config_file, "w")
+    cluster_config_file.write(rendered_tmpl)
+    cluster_config_file.close()
+    return dind_cluster_config_file, dind_kube_config_file
+
+
+def get_rke_config_template():
+    dind_cluster_config_j2 = """
+---
+nodes:{% for node in random_nodes %}
+  - address: {{ node }}
+    user: docker
+    role:
+    - controlplane
+    - worker
+    - etcd{% endfor %}
+"""
+    return dind_cluster_config_j2
+
+
+def build_dind(cluster_config_file):
```

Comment:
we should consider renaming the methods to reflect their purpose. create_cluster, remove_cluster

Reply:
will change it